static/app/views/explore/utils.tsx

```diff
@@ -58,7 +59,9 @@ export function getExploreUrl({
   );
 }
 
-export function combineConfidenceForSeries(series: Series[]): Confidence {
+export function combineConfidenceForSeries(
+  series: Array<Pick<TimeseriesData, 'confidence'>>
```

Comment:
Would it better to just do `TimeseriesData[]` here as the function implies that an entire series is the parameter?

 Or is the `Pick` here for backwards compatibility with `Series`? If that's case we could also consider `TimseriesData[] | Series[]`

Reply:
Yep exactly, for backward compatibility

Reply:
Union makes more sense to me, because with the Pick, I believe passing in any array of objects, where the object has confidence as a property, would work.

Reply:
And if we removed Series eventually, it's easier to find this.